Doc/whatsnew/3.10.rst

```diff
@@ -1353,3 +1353,8 @@ Removed
   Python already implicitly installs signal handlers: see
   :c:member:`PyConfig.install_signal_handlers`.
   (Contributed by Victor Stinner in :issue:`41713`.)
+
+* Remove the ``PyAST_Validate()`` function. It is no longer possible to build a
+  AST object (``mod_ty`` type) with the public C API. The function was already
+  excluded from the limited C API (:pep:`384`).
```

Comment:
Hm, `pep-384` doesn't mention the ast.h. Can we move the capi directly when this C API is not a limited C API?

Reply:
I just mentioned PEP 384 to explain what is the limited C API.

PyAST_Validate() definition is inside:

```
#ifndef Py_LIMITED_API
...
#endif /* !Py_LIMITED_API */
```

Reply:
"Can we move the capi directly when this C API is not a limited C API?" No. I only removed the function since it cannot be used anymore since Python 3.10: " It is no longer possible to build a AST object (``mod_ty`` type) with the public C API."

Reply:
OK, Thanks. got it.